CONTRIBUTING.md

```diff
@@ -33,23 +69,23 @@ diverse, inclusive, and healthy community.
 Examples of behavior that contributes to a positive environment for the
 project include:
 
-* Demonstrating empathy and kindness toward other people
-* Being respectful of differing opinions, viewpoints, and experiences
-* Giving and gracefully accepting constructive feedback
-* Accepting responsibility and apologizing to those affected by our mistakes,
+- Demonstrating empathy and kindness toward other people
```

Comment:
I didn't want to touch this section but there was a Github Actions markdownlint workflow blocking up things if I didn't correct it. 🤷‍♂️